CleanSlate/common/scripted_effects/03_sainthood_effects.txt

```diff
@@ -19,14 +19,7 @@ deciding_upon_saint_effect = {
 			# Have to be a Christian denomination with a potential Religious head, and have been singled out for Beatification
 			event_target:saint_person = {
 				OR = {
-					religion = catholic
-					religion = fraticelli
-					religion = orthodox
-					religion = iconoclast
-					religion = miaphysite
-					religion = monophysite
-					religion = monothelite
-					religion = paulician
+					religion_group = christian
```

Comment:
I suspect that this change is incorrect. There are six Christian religions unaccounted for in the original that are now caught by the `religion_group` specifically: `cathar`, `lollard`, `waldensian`, `bogomilist`, `nestorian`, and `messalian`